src/llama-model.cpp

```diff
@@ -555,6 +555,14 @@ void llama_model::load_hparams(llama_model_loader & ml) {
 
     ml.get_key_or_arr(LLM_KV_ATTENTION_HEAD_COUNT_KV, hparams.n_head_kv_arr, hparams.n_layer, false);
 
+    ml.get_key(LLM_KV_ATTENTION_SLIDING_WINDOW, hparams.n_swa, false);
+    if (hparams.n_swa > 0) {
+        hparams.swa_type = LLAMA_SWA_TYPE_STANDARD;
```

Comment:
I don't think we can assume anything about `swa_type` type here. Please remove this line

Reply:
I don't get it. how does swa not get activate?

`swa_type` should be set per-model. Please remove this line.